apps/ios/Shared/ContentView.swift

```diff
@@ -73,8 +73,7 @@ struct ContentView: View {
                     showSetPasscode = false
                     alertManager.showAlert(laPasscodeNotSetAlert())
                 }
-            }
-            if chatModel.chatDbStatus == nil {
+            } else if chatModel.chatDbStatus == nil && AppChatState.shared.value != .stopped {
```

Comment:
this or something else breaks authentication somehow.. try to fail authentication and app will show initialization view instead of unlock button

Reply:
this happens if you close the app, not just put to background

Reply:
https://github.com/simplex-chat/simplex-chat/pull/3640/commits/4b1b325896d27801be8fc78540d0f3c5750a877d


Looks like without another flag is impossible to distinguish states properly to not lose initialization view when it's needed. Now it should be ok